spec/models/item_spec.rb

```diff
@@ -14,8 +14,8 @@
       expect(item.status).to eq("clearanced")
     end
 
-    it "should set the price_sold as 75% of the wholesale_price" do
-      expect(item.price_sold).to eq(BigDecimal.new(wholesale_price) * BigDecimal.new("0.75"))
+    it "should set the price_sold as a percentage of the wholesale_price" do
```

Comment:
If we don't expect the value of `CLEARANCE_PERCENTAGE` to vary, it might be more clear to interpolate it in this test case's name.